test/airdrop-test.js

```diff
@@ -0,0 +1,52 @@
+const { ethers } = require("hardhat");
+const { use, expect } = require("chai");
+const { solidity } = require("ethereum-waffle");
+const { accounts, contract } = require('@openzeppelin/test-environment');
+
+use(solidity);
+
+describe("NestCoinFilmHouse", function () {
+  let multiTransferTokenEqual;
+
+  // quick fix to let gas reporter fetch data from gas station & coinmarketcap
+  before((done) => {
+    setTimeout(done, 2000);
+  });
+
+  describe("MultiTransferTokenEqual", function () {
+    it("Should deploy token contract", async function() 
```

Comment:
Please create a before each function that would run before every it function. The before each function should create get and deploy your contract. The contracts,accounts etc that would be used across different it functions should be global variables (i.e they should be declared outside the describe function).

E.g of what the before would look like
`beforeEach(async () => {
    accounts = await ethers.getSigners();
    owner = accounts[0];
    contractFactory = await ethers.getContractFactory("VotingContract");
    contestantNamesInBytes = await createBytes();
    votingContract = await contractFactory.deploy(contestantNamesInBytes);
    await votingContract.deployed();
  });
`